content/v2/secondary-dns.markdown

```diff
@@ -0,0 +1,341 @@
+---
+title: Secondary DNS API | DNSimple API v2
+excerpt: This page documents the DNSimple secondary DNS API v2.
+---
+
+# Secondary DNS API
+
+* TOC
+{:toc}
+
+
```

Comment:
```suggestion
<note>
<h4>This endpoint is currently in Public Preview. During the Preview period changes may occurr at any time.</h4>
</note>

```

Reply:
@weppos @aeden How do we want to say this? Should we treat the API as private beta?

Reply:
Yes, you can replace "Public Preview" with "Private Beta". Also, "occur" is misspelled in that suggestion above. 😄

Reply:
😄 we should change the spelling at https://github.com/dnsimple/dnsimple-developer/blob/main/layouts/v2-preview.html

Reply:
I think we can drop `v2-preview`